io/plugins/eu.esdihumboldt.hale.io.shp/src/eu/esdihumboldt/hale/io/shp/writer/ShapefileInstanceWriter.java

```diff
@@ -226,13 +227,15 @@ private Map<String, Map<String, SimpleFeatureType>> createFeatureType(
 				// present.
 			}
 		}
-		for (String localPart : missingGeomsForSchemas) {
-			reporter.error(
-					"Cannot create Shapefile for Schema: " + localPart + " as no Geometry found!!");
-		}
+
 		// create SimpleFeatureType from SimpleFeatureTypeBuilder.
 		for (Entry<String, Map<String, SimpleFeatureTypeBuilder>> schemaEntry : schemaBuilderMap
 				.entrySet()) {
+
+			if (missingGeomsForSchemas.contains(schemaEntry.getKey())) {
+				continue;
```

Comment:
Maybe log a warning in the reporter that the type was skipped? That would IMHO be better than just silently skipping it.